_includes/disqus_comments.html

```diff
@@ -11,4 +11,4 @@
     (d.head || d.body).appendChild(s);
   })();
 </script>
-<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>
\ No newline at end of file
+<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>
```

Comment:
Any reason for this change?

Reply:
Is this required, if not, can you update the commit to remove this change?

Reply:
I didn't touch this code, but it says it's changed 😢

Reply:
Can you update such that this is not reflected here?